src/Bicep.Core/Semantics/DescriptionHelper.cs

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using Bicep.Core.Json;
+using System.IO;
+using Bicep.Core.Extensions;
+using Bicep.Core.Syntax;
+using System.Linq;
+using Bicep.Core.Semantics.Namespaces;
+using Bicep.Core.Modules;
+using System.Collections.Immutable;
+
+namespace Bicep.Core.Semantics
+{
+    public static class DescriptionHelper
+    {
+        private const string Description = LanguageConstants.MetadataDescriptionPropertyName;
```

Comment:
`Description` is used in some places while `LanguageConstants.MetadataDescriptionPropertyName` is used in other places. Why not use the `LanguageConstants...` directly for the usages? I suppose `Description` is shorter and reads more nicely in the interpolation brackets but the naming might also imply it's the description of something rather than a property key for an object. Another benefit of direct usage of the `LanguageConstants` constant is when finding usages in an IDE, the listing will be exactly where it's used. Adding an alias to this can slow down usage analysis.

Reply:
938592db75c80f52259936b31e12b4c8c557bfdd